tests/bug-detectors/command-injection.test.js

```diff
@@ -29,6 +29,7 @@ describe("Command injection", () => {
 
 	it("exec with EVIL command", () => {
 		const fuzzTest = new FuzzTestBuilder()
+			.runs(0)
```

Comment:
Moved the default `0` runs into the builder setup to set as few parameters as possible for the actual test invocation. The builders could be extracted into common ones quite easily, but I didn't want to spend more time on that for now.